### ISSUE TITLE
The OpenVZ example will give a Not Implemented error

### ISSUE THREAD

**Issue by Ciev** [telcat/vagrant-proxmox#27]
Hi, 

I was trying to do both examples openVZ and KVM and the process will only work with KVM... after setting up everything to work with openVZ I get the following error:

user@user-All-Series:~/vagrant-proxmox-master/dummy_box$ vagrant up --provider=proxmox
Bringing machine 'box' up with 'proxmox' provider...
==> box: Creating the virtual machine...
**Unable to communicate with proxmox server:**

**Unable to create the virtual machine!**

**Cause: VagrantPlugins::Proxmox::ApiError::NotImplemented**

my Vagrant fie is configured as:

Vagrant.configure('2') do |config|

    config.vm.provider :proxmox do |proxmox|
        proxmox.endpoint = 'https://192.168.10.93:8006/api2/json'
        proxmox.user_name = 'admin@pam'
        proxmox.password = 'pass'
        proxmox.vm_id_range = 900..910
        proxmox.vm_name_prefix = 'vg_pm_dev_'
        proxmox.openvz_os_template = 'local:iso/vagrant-proxmox-ubuntu-12.tar.gz'
        proxmox.vm_type = :openvz
        proxmox.vm_memory = 1024
    end

    config.vm.define :box, primary: true do |box|
        box.vm.box = 'dummy'
        box.vm.network :public_network, ip: '192.168.10.94'
    end
 
end 

Basically the example template.. 
I'm using
   * Vagrant 1.8.1
   * ruby 2.2.4p230

I have no idea what went wrong... If you can give me any hint, I'll appreciated.

Best

**Comment by devekko** [comment]
on Proxmox 4, its LXC now, not OpenVZ

I'm also trying to work this out

I blogged about my approach here
http://niccolox.org/compile-custom-vagrant-plugin-for-proxmox-provider/
